Sources/PointFree/Account/UpdateProfile.swift

```diff
@@ -81,7 +82,7 @@ let updateProfileMiddleware =
         updateFlash = flash(.notice, "We’ve updated your profile!")
       }
 
-      let updateCustomerExtraInvoiceInfo = zip2(
+      let updateCustomerExtraInvoiceInfo = zip(
```

Comment:
We only have a single overload on Optional...if this is an issue we can go back to numbering.

Reply:
we should also prob just bring in overture. I'd like to start using it more anyway. I'll add a Trello card